CHANGES.rst

```diff
@@ -1,11 +1,12 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-09)
 --------------------------
 
 - Allows restarting of yadage workflows.
```

Comment:
Need to respect Adds/Fixes/Changes triad to indicate new features, bug fixes, and improvements.  So something like:

- Adds support for specifying `htcondor_max_runtime` and `htcondor_accounting_group` for HTCondor compute backend jobs.
- Fixes restarting of Yadage workflows.
- Changes Continuous Integration from Travis to GitHub Actions.

BTW the latter is more interesting for developers; I'm not even sure whether we should mention it, since it does not change the `r-w-e-yadage` codebase at all...